src/App.tsx

```diff
@@ -680,13 +659,19 @@ export default function App() {
   return (
     <StyledEngineProvider injectFirst>
       <CssBaseline />
-      <UserProvider data={{ user, reloadUser }}>
-        {selectedOrganization && (
-          <OrganizationProvider data={{ selectedOrganization, setSelectedOrganization, organizations, reloadData }}>
-            <LocalizationProvider locale={locale}>{getContent()}</LocalizationProvider>
-          </OrganizationProvider>
-        )}
-      </UserProvider>
+      {selectedOrganization && (
+        <OrganizationProvider data={{ selectedOrganization, setSelectedOrganization, organizations, reloadData }}>
+          <LocalizationProvider locale={locale}>{getContent()}</LocalizationProvider>
+        </OrganizationProvider>
+      )}
     </StyledEngineProvider>
   );
 }
+
+export default function App(): JSX.Element {
+  return (
+    <UserProvider>
```

Comment:
We can add the OrgProvider below this.. it doesn't have to be a blocking provider.
AppBase can continue to do it's thing based on the selectedOrg,, we can clean that up later.
We could theoretically load strings as part of the UserProvider and expose the data. Localization provider could just re-expose it for semantics purposes, maybe 🤷 
Just some thoughts.

Reply:
We could refactor this some more by introducing a 'Bootstrap' layer - this would be blocking since it's bootstrapping the app. What happens inside that is up to us. This way we can separate out providers and blocking semantics.
```
<App>
  <Bootstrap>
      <AppBase />
  </Bootstrap>

Boostrap:
   initialize the various providers maybe.
   block on required info but not on optional info
```

Fwiw, I noticed that we - today - block the app until we have data back from the orgs fetch. App requires knowledge of whether user has orgs no no-orgs - so it can render the correct path (no-orgs-landing-page versus full-fledged-app).

Reply:
Ah, makes sense